File: EPS FrontEnd/EPS/src/app/models/Laboratory/laboratory-request.ts
import { Laboratory } from './laboratory';

///Parametros de peticion de laboratorio
export class LaboratoryRequest
{
  /// <summary>
  /// Codigo
  /// </summary>
  UserCode: string;
  /// <summary>
  /// Historia
  /// </summary>
  History: number;
  /// <summary>
  /// Medicamento
  /// </summary>
  Laboratory: Laboratory;
}

File: EPS BackEnd/EPS.WebServices/EPS.Common.Models/Laboratory/LaboratoryRequest.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace EPS.Common.Models
{
    public class LaboratoryRequest
    {
        public string UserCode { get; set; }
        /// <summary>
        /// Historia
        /// </summary>
        public int History { get; set; }
        /// <summary>
        /// Medicamento
        /// </summary>
        public Laboratory Laboratory { get; set; }
    }
}
